test/reporters/tap.js

```diff
@@ -146,3 +146,39 @@ test('results does not show skipped tests if there are none', function (t) {
 	t.is(actualOutput, expectedOutput);
 	t.end();
 });
+
+test('todo test', function (t) {
+	var reporter = tapReporter();
+
+	var actualOutput = reporter.test({
+		title: 'should think about doing this',
+		passed: false,
+		todo: true
```

Comment:
If we're sticking to having both `skipped` and `todo` in the metadata then you'll have to add `skipped: true` here.